README.md

```diff
@@ -4,7 +4,7 @@
 
 > Project Microverse Trials Code Review
 
-![screenshot](./app_screenshot.png)
+![screenshot](norman.png)
```

Comment:
* This is an example inline comment. 🙂

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. 😉

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.